### ISSUE TITLE
HTML Element must have LANG attribute

### ISSUE THREAD

**Issue by jmcglone** [mlibrary/heliotrope#615]
Fulcrum app is missing html lang attribute

The <html> element does not have a lang attribute
`.js.wf-droidserif-n4-active.wf-droidserif-i4-active.wf-freightsanspro-n4-active.wf-droidserif-i7-active.wf-freightsanspro-n3-active.wf-freightsanspro-i4-active.wf-droidserif-n7-active.wf-freightsanspro-n6-active.wf-freightsanspro-i6-active.wf-active`
`<html class="js wf-droidserif-n4-active wf-droidserif-i4-active wf-freightsanspro-n4-active wf-droidserif-i7-active wf-freightsanspro-n3-active wf-freightsanspro-i4-active wf-droidserif-n7-active wf-freightsanspro-n6-active wf-freightsanspro-i6-active wf-active">`
